glossary.md

```diff
@@ -135,23 +243,31 @@ Quota Limiter is an experimental feature introduced in TiDB v6.0.0. If the machi
 
 Raft Engine is an embedded persistent storage engine with a log-structured design. It is built for TiKV to store multi-Raft logs. Since v5.4, TiDB supports using Raft Engine as the log storage engine. For details, see [Raft Engine](/tikv-configuration-file.md#raft-engine).
 
+### Region split
+
+Regions are generated as data writes increase. The process of splitting is called Region split.
```

Comment:
```suggestion
### Region Split

A region in a TiKV cluster is not divided at the beginning, but is gradually split as data is written to it. The process is called Region split.
```

Reply:
Not sure if it is clear enough to a reader that this is a TiKV data region and not an AWS region etc